docs/en/essentials/getting-started.md

```diff
@@ -64,6 +64,27 @@ const app = new Vue({
 // Now the app has started!
 ```
 
+By injecting the router, we get access to it as `this.$router` as well as the current route as `this.$route` inside of any component:
+
+```js
+// Home.vue
+export default {
+  computed: {
+    params () {
+      // We will see what params are shortly
+      return this.$route.params
+    }
+  },
+  methods: {
+    someMethod () {
+      // Redirecting to another route
+      this.$router.push('/redirect')
+    }
+}
+```
+
+Thoroughly the docs, we will often use the `router` instance. Keep in mind that `this.$router` is exactly the same as using `router`. The reason we use `this.$router` is because we don't want to import the router in every single component that needs to manipulate routing.
```

Comment:
Hmm, I'm actually wondering now if this paragraph is even necessary. I feel like the idea that `router === this.$router` is made clear by:

> By injecting the router, we get access to it as `this.$router`